esp8266/scripts/main.py

```diff
@@ -0,0 +1 @@
+# This script is run on boot
```

Comment:
Well, if there is no way to run code on boot, the usage is limited to having the board connected to a PC and having to reload the script on every power cycle. This way, the user can store a script to be run on boot, which makes the chip completely standalone.